### ISSUE TITLE
Quesiton for code

### ISSUE THREAD

**Issue by xutting123** [QinbinLi/MOON#1]
Hi, I have read your paper and code and found it to be an interesting work!

I have a question. In your code, 

>  loss2 = mu * criterion(logits, labels)    # (main.py line311).

I know it is uesd to caculate the con_loss (Eq.3 in your paper), but why it is implemented by cross-entropy loss with labels (zeros tensor)?

**Comment by QinbinLi** [comment]
Hi @xutting123 ,

The contrastive loss can be computed by the cross-entropy loss. In the [formula](https://pytorch.org/docs/stable/generated/torch.nn.CrossEntropyLoss.html), by setting labels to 0, x[class] (i.e., x[0]) is the positive pair part.
